aws-py-oidc-provider/__main__.py

```diff
@@ -0,0 +1,70 @@
+import pulumi
+from pulumi_aws import iam
+import requests
+import subprocess
+import OpenSSL
+import json
+
+audience = "" # Provide the name of your Pulumi Organization
+oidc_idp_url = 'https://api.pulumi.com/oidc'
+base_url = 'api.pulumi.com/oidc'
+
+# Obtain the OIDC IdP URL and form the configuration document URL
+print("Forming configuration document URL...")
+configuration_url = f'{oidc_idp_url}/.well-known/openid-configuration'
+
+# Locate "jwks_uri" and extract the domain name
+print("Extracting domain name from jwks_uri...")
```

Comment:
This should probably be modeled as a resource using the Command provider: https://www.pulumi.com/registry/packages/command/api-docs/local/command/

Reply:
I don't really understand the ask here. Can we sync on this and the TLS comment?